build/protobuf.mk

```diff
@@ -44,13 +44,21 @@ CPP_PROTOS := $(filter %api.proto %data.proto %internal.proto %config.proto %err
 CPP_HEADERS := $(subst ./,$(ENGINE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.h)) $(subst $(GOGOPROTO_ROOT),$(ENGINE_ROOT),$(GOGOPROTO_PROTO:%.proto=%.pb.h))
 CPP_SOURCES := $(subst ./,$(ENGINE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.cc)) $(subst $(GOGOPROTO_ROOT),$(ENGINE_ROOT),$(GOGOPROTO_PROTO:%.proto=%.pb.cc))
 
+ENGINE_PROTOS := $(sort $(shell find $(ENGINE_ROOT) -name *.proto -type f))
+ENGINE_CPP_HEADERS := $(ENGINE_PROTOS:%.proto=%.pb.h)
+ENGINE_CPP_SOURCES := $(ENGINE_PROTOS:%.proto=%.pb.cc)
+
 .PHONY: protos
-protos: $(GO_SOURCES) $(CPP_HEADERS) $(CPP_SOURCES)
+protos: $(GO_SOURCES) $(CPP_HEADERS) $(CPP_SOURCES) $(ENGINE_CPP_HEADERS) $(ENGINE_CPP_SOURCES)
 
 $(GO_SOURCES): $(PROTOC) $(GO_PROTOS) $(GOGOPROTO_PROTO) $(PROTOC_PLUGIN)
 	find $(REPO_ROOT) -not -path '*/.*' -name *.pb.go | xargs rm
 	for dir in $(sort $(dir $(GO_PROTOS))); do \
-	  $(PROTOC) -I.:$(GOGOPROTO_PATH) --plugin=$(PROTOC_PLUGIN) --$(PLUGIN_SUFFIX)_out=$(ORG_ROOT) $$dir/*.proto; \
+	  $(PROTOC) -I.:$(GOGOPROTO_PATH) --plugin=$(PROTOC_PLUGIN) --$(PLUGIN_SUFFIX)_out=import_prefix=github.com/cockroachdb/:$(ORG_ROOT) $$dir/*.proto; \
+	  sed -i.bak 's/import math "github\.com\/cockroachdb\/math"//g' $$dir/*.pb.go; \
```

Comment:
It doesn't look like they add `import_prefix` to the `"math"` import (https://github.com/golang/protobuf/blob/master/protoc-gen-go/generator/generator.go#L1153)